stubs/portpicker/portpicker.pyi

```diff
@@ -0,0 +1,18 @@
+from typing_extensions import TypeAlias
+
```

Comment:
```suggestion

__all__ = ('bind', 'is_port_free', 'pick_unused_port', 'return_port',
           'add_reserved_port', 'get_port_from_port_server')

```

Reply:
Good catch. Cross-referencing python/mypy#10321: stubgen should do this for us.